core/scheduler/src/main/scala/org/apache/openwhisk/core/scheduler/queue/QueueManager.scala

```diff
@@ -397,7 +400,7 @@ class QueueManager(
       logging.warn(
         this,
         s"[${msg.activationId}] the activation message has not been scheduled for ${queueManagerConfig.maxSchedulingTime.toSeconds} sec")
-      completeErrorActivation(msg, "The activation has not been processed")
+      completeErrorActivation(msg, "The activation has not been processed: too old activation is arrived.")
```

Comment:
Not sure this effectively describes the case.
This is the case that activations arrived after the max scheduling wait time.
For example, this can happen when there was a failure in Kafka so activations could not be delivered and just stored in it. When Kafka becomes available again, it will start delivering activations.
But if it took so much time to restore Kafka such as 1 hour, it will send too old(1 hour-old) activations.
Also, if there were many activations stored in Kafka before the failure, it would cause a thundering herd by sending them all at the same time. So we complete them with an error.